lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+        ],
+        notaccept: [
+          'POL-P3', # Facebook is not accepting messages from your mail server. This will persist for 4 to 8 hours.
+          'POL-P4', # Facebook is not accepting messages from your mail server. This will persist for 24 to 48 hours.
+          'POL-T1', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 1 to 2 hours.
+          'POL-T2', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 4 to 8 hours.
+          'POL-T3', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 24 to 48 hours.
+        ],
+        rejected: [
+          'DNS-P1', # Your SMTP MAIL FROM domain does not exist.
+          'DNS-P2', # Your SMTP MAIL FROM domain does not have an MX record.
+          'DNS-T1', # Your SMTP MAIL FROM domain exists but does not currently resolve.
+          'DNS-P3', # Your mail server does not have a reverse DNS record.
+          'DNS-T2', # You mail server's reverse DNS record does not currently resolve.
+        ],
+        systemerror: [
+          'CON-T1', # Facebook's mail server currently has too many connections open to allow another one.
+        ],
+        toomanyconn: [
+          'CON-T3', # Your mail server has opened too many new connections to Facebook's mail servers in a short period of time.
+        ],
+        suspend: [
+          'RCP-T4', # The attempted recipient address is currently deactivated. The user may or may not reactivate it.
+        ],
+        undefined: [
+          'RCP-T1', # The attempted recipient address is not currently available due to an internal system issue. This is a temporary condition.
+          'MSG-T1', # The number of recipients on the message exceeds Facebook's allowed maximum.
+          'CON-T2', # Your mail server currently has too many connections open to Facebook's mail servers.
+          'CON-T4', # Your mail server has exceeded the maximum number of recipients for its current connection.
+        ],
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Facebook: https://www.facebook.com'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return []; end
+      def pattern;     return Re0; end
+
+      # Parse bounce messages from Facebook
+      # @param         [Hash] mhead       Message headers of a bounce email
+      # @options mhead [String] from      From header
+      # @options mhead [String] date      Date header
+      # @options mhead [String] subject   Subject header
+      # @options mhead [Array]  received  Received headers
+      # @options mhead [String] others    Other required headers
+      # @param         [String] mbody     Message body of a bounce email
+      # @return        [Hash, Nil]        Bounce data list and message/rfc822
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->